### ISSUE TITLE
Fatal error: Uncaught Error: Class 'Puc_v4_Factory' not found (Composer Autoload)

### ISSUE THREAD

**Issue by connerbw** [YahnisElsts/plugin-update-checker#321]
Docs say:

> If you're using the Composer autoloader, you don't need to explicitly require the library.

With code snippet:

```php
$myUpdateChecker = Puc_v4_Factory::buildUpdateChecker(
	'http://example.com/path/to/details.json',
	__FILE__, //Full path to the main plugin file or functions.php.
	'unique-plugin-or-theme-slug'
);
```

Latest 4.8 version breaks code. 

Error goes away if I replace with `\Puc_v4p8_Factory::buildUpdateChecker()`

![image](https://user-images.githubusercontent.com/812192/66345080-4a37b900-e91d-11e9-97ce-aa17d23232ac.png)

**Comment by YahnisElsts** [comment]
Thank you for the report. This should now be fixed in the latest release (4.8.1).

**Comment by connerbw** [comment]
Thanks for the fix. Works for me.
